docroot/modules/custom/uiowa_hours/src/HoursApi.php

```diff
@@ -167,28 +169,69 @@ public function getResources($group) {
    * @return array
    *   An array of hours.
    */
-  public function getHours($resource, $params) {
-    $data = $this->request('GET', "$this->group/$resource", $params);
-    $date = $params['start'];
-    $key = date('Ymd', strtotime($date));
-    $markup = 'No hours information available.';
-
-    if ($data->$key) {
-      $markup = '';
-      $resource_hours = $data->$key;
-      // @todo If there are multiple instances then this needs better formatting.
-      foreach ($resource_hours as $time) {
-        $start = date('g:ia', strtotime($time->startHour));
-        $end = '00:00:00' ? strtotime($time->endHour . ', +1 day') : strtotime($time->endHour);
-        $end = date('g:ia', $end);
-        $markup .= $time->summary . ' ' . $start . ' - ' . $end;
+  public function getHours($resource, $start = 'today', $end = 'today') {
+    $data = $this->request('GET', "$this->group/$resource", [
+      'start' => date('m/d/Y', strtotime($start)),
+      'end' => ($end == $start) ? '' : date('m/d/Y', strtotime($end)),
+    ]);
+
+    $render = [
+      '#type' => 'container',
+      '#attributes' => [
+        'class' => [
+          'uiowa-hours-container',
+        ],
+      ],
+    ];
+
+    if (empty($data)) {
+      $render['closed'] = [
+        '#type' => 'tag',
+        '#tag' => 'span',
+        '#attributes' => [
+          'class' => [
+            'uiowa-hours-status',
+            'badge',
+            'badge--orange',
+          ],
+        ],
+        '#markup' => $this->t('Closed Today'),
+      ];
+    }
+    else {
+      // The v2 API indexes events by a string in Ymd format, e.g. 20211209.
+      foreach ($data as $key => $date) {
+        $render['hours'][$key] = [
+          '#type' => 'container',
+          '#attributes' => [
+            'class' => [
+              'uiowa-hours',
+            ],
+          ],
+          'date' => [
+            '#markup' => $this->t('<h6>@date</h6>', [
```

Comment:
This should change to another element. Wasn't sure what to use.

Reply:
when in doubt, i've been using `'#type' => 'item',`

Reply:
Well I mean I think a `h6` is not ideal. Maybe just a `p` with styling?

Reply:
this is now just a string and pulled in as card title